### ISSUE TITLE
Invariant Violation: addComponentAsRefTo

### ISSUE THREAD

**Issue by luckyraul** [fmoo/react-typeahead#57]
Hi, I am trying to add your module into my project and got. 
```Invariant Violation: addComponentAsRefTo(...)```

Am I doing something wrong?
```
React.render(
    <App />,
    document.getElementById('service')
);
```
app
```
var App = React.createClass({

    render: function () {
        return (
            <div className="app">
                <SearchBlock />
            </div>
        );
    }
});
```
SeachBlock
```
var React = require('react');
var Calendar = require('react-widgets').Calendar;
var Typeahead = require('react-typeahead').Typeahead;
module.exports = React.createClass({
    render: function () {
        return (
            <div className="a">
                <Typeahead options={["janet", "pete", "may", "jason"]}/>
                <Calendar />
            </div>
        );
    }
});
```

**Comment by thehuey** [comment]
I don't think you're doing anything wrong.  Can you take out the <Calendar
/> element?  I did a quick test on react-typeahead 1.0.3 and it worked fine
with the above example without the Calendar element

**Comment by seansfkelley** [comment]
I saw this after updating my React version to fix #56. In my case it's because I had two version of React (one from my environment, one from react-typeahead) and they were conflicting. Updating my versions to match and deleting the React module from react-typeahead fixed it up. Originally from [this StackOverflow post](http://stackoverflow.com/questions/28519287/what-does-only-a-reactowner-can-have-refs-mean).

**Comment by luckyraul** [comment]
Without Calendar - the same problem

Project React.version = '0.13.1';
Yours = '0.13.1';
